specs/decimal/decimal.md

```diff
@@ -45,7 +57,152 @@ data structure may be re-expressed in each language using language-specific cons
 defines the `DecimalValue` structure as the Win32 `DECIMAL` definition syntax is incompatible with
```

Comment:
Might link to the set of APIs this is related to - the `VarDec*` helpers, like https://learn.microsoft.com/en-us/windows/win32/api/oleauto/nf-oleauto-vardecfromstr ?